frontend/src/components/custom-tools/header/Header.jsx

```diff
@@ -199,6 +227,21 @@ function Header({
           toolDetails={toolDetails}
         />
       </div>
+
+      {confirmModalVisible && (
+        <Modal
+          onOk={handleConfirmForceExport} // Pass the confirm action
+          onCancel={() => setConfirmModalVisible(false)} // Close the modal on cancel
+          open={confirmModalVisible}
+          title="Are you sure"
+          okText="Force Export"
+          centered
+        >
+          Unable to export tool. Some Prompt(s) were not run. Please run them
```

Comment:
```suggestion
          Unable to export tool. Some prompt(s) were not run. Please run them
```

Reply:
As a future improvement, we could also try to list the prompts which were not run

Reply:
@chandrasekharan-zipstack We're not throwing as error but as warning. Can check the exception we set code as warning. In future we can set few more codes like warning, info like that and instead of showing only success and failure popup we can have few more options there.